tensorflow_probability/python/mcmc/dual_averaging_step_size_adaptation.py

```diff
@@ -0,0 +1,299 @@
+"""DualAveragingStepSizeAdaptation TransitionKernel."""
```

Comment:
Can you also add a unit test? You should be able to use SimpleStepSize as an example.